src/fetch.cpp

```diff
@@ -89,6 +89,11 @@ namespace mamba
             }
         }
 
+        if (Context::instance().ssl_no_revoke || std::getenv("MAMBA_SSL_NO_REVOKE") != nullptr)
```

Comment:
should we check the value of the env variable?

Reply:
Sure thing.  Any preference on what values to support for it though (true, 1)?

Reply:
I am voting for `0` == false, everything else true?

Reply:
Sounds good to me.